lib/rust/ensogl/core/src/display/scene.rs

```diff
@@ -470,7 +470,7 @@ impl Dom {
 // === DomLayers ===
 // =================
 
-/// DOM DomLayers of the scene. It contains a 2 CSS 3D layers and a canvas layer in the middle. The
+/// DOM DomLayers of the scene. It contains a 3 CSS 3D layers and a canvas layer in the middle. The
```

Comment:
I would just remove the number and write that it contains several layers instead :P